backend/de.metas.printing/de.metas.printing.base/src/main/java/de/metas/printing/api/IPrintingQueueBL.java

```diff
@@ -110,4 +110,6 @@
 	void setPrintoutForOtherUsers(I_C_Printing_Queue item, Set<Integer> userToPrintIds);
 
 	PrinterRoutingsQuery createPrinterRoutingsQueryForItem(I_C_Printing_Queue item);
+
+	void setProcessedAndSave(@NonNull I_C_Printing_Queue item);
```

Comment:
imho ok since we don't have IPrintingQueueDAO and i think it makes no sence to add one just for this method